components/coachmark/stories/coachmark.stories.js

```diff
@@ -83,21 +135,41 @@ Standard.parameters = {
 	chromatic: {
 		disableSnapshot: true,
 	},
+	docs: {
+		story: {
+			height: "450px",
+		},
+	},
```

Comment:
I think there are 3 stories with additional parameters for the 450px story heights. I vote we just delete all of these repetitions, and update the default `docs.story.height: "300px"` to the 450. Then we only have to redefine that one 725px story. 🥳